libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -269,6 +269,19 @@ func (m *Manager) Apply(pid int) error {
 			newProp("CPUShares", uint64(c.Resources.CpuShares)))
 	}
 
+	// cpu.cfs_quota_us and cpu.cfs_period_us are controlled by systemd.
+	// The percentage cpuQuota refered specifies how much CPU time the unit
+	// shall get at maximum, relative to the total CPU time available on one CPU.
+	// The percentage should be greater than 1%.
+	if c.Resources.CpuQuota != 0 && c.Resources.CpuPeriod != 0 {
+		cpuQuota := c.Resources.CpuQuota * 100 / c.Resources.CpuPeriod
+		if cpuQuota == 0 {
+			return fmt.Errorf("cpu_quota divided by cpu_period must be greater than 0.01")
```

Comment:
Because the systemd can not accept that the percentage(cpu_quota divided by cpu_period) is less than 0.01. For example,  if the user let CpuQuota be 10 and CpuPeriod be 10000, the cpu.cfs_quota_us will be -1, not 10. So, I think it is necessary to warn the user that the percentage must be greater than 0.01.